app/src/components/integrations/modals/addLdapIntegrationModal/twoSteps/twoStepsContent/twoStepsContent.jsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Button } from '@reportportal/ui-kit';
+import { Scrollbars } from 'rc-scrollbars';
+import { reduxForm } from 'redux-form';
+import { INTEGRATION_FORM } from 'components/integrations/elements';
+import { LdapFormFields } from 'components/integrations/integrationProviders';
+import { useWindowResize } from 'common/hooks';
+import styles from './twoStepsContent.scss';
+
+const cx = classNames.bind(styles);
+
+const MODAL_MAX_RATIO = 0.9;
+const MODAL_HEADER_HEIGHT = 32 + 24;
+const MODAL_FOOTER_HEIGHT = 36 + 16 + 16;
+const MODAL_LAYOUT_PADDING = 32 * 2;
+
+const TwoStepsContent = ({
+  steps = [],
+  handleSubmit,
+  onSubmit,
+  initialize,
+  change,
+  data,
+  stepNumber,
+  updateMetaData,
+}) => {
+  const { customProps, initialData } = data;
+  const windowSize = useWindowResize();
+  const windowHeight = windowSize.height;
+  const modalMaxHeight = windowHeight * MODAL_MAX_RATIO;
+  const contentMaxHeight =
+    modalMaxHeight - MODAL_LAYOUT_PADDING - MODAL_FOOTER_HEIGHT - MODAL_HEADER_HEIGHT;
+  return (
+    <div className={cx('two-steps-content')}>
+      <div className={cx('step-list')}>
+        {steps.map(({ title, index, onClick, active }) => (
+          <Button key={index} className={cx('step', { active })} onClick={onClick} variant={'text'}>
+            {`${index}. ${title}`}
+          </Button>
+        ))}
+      </div>
+      <Scrollbars
```

Comment:
We use it in the modal in ui-kit. I thought we were planning to use it in service-ui.

Reply:
It was used temporary because right now we don't have a separate scroll component in ui-kit.
Please leave a TODO comment that it will be replaced with new Scroll component in the future.